formats/json/commonMain/src/kotlinx/serialization/json/JsonAnnotations.kt

```diff
@@ -74,3 +74,36 @@ public annotation class JsonNames(vararg val names: String)
 @Target(AnnotationTarget.CLASS)
 @ExperimentalSerializationApi
 public annotation class JsonClassDiscriminator(val discriminator: String)
+
+
+/**
+ * Specifies whether encounters of unknown properties (i.e., properties not declared in the class) in the input JSON
+ * should be ignored instead of throwing [SerializationException].
+ *
+ * With this annotation, it is possible to allow unknown properties for annotated classes, while
+ * general [Json.decodeFromString] would still reject them for everything else.
```

Comment:
Perhaps I'm exaggerating, but won't such a description give the user the impression that this annotation only works with `decodeFromString`, but not with `decodeFromJsonElement` or `decodeFromStream`?

Reply:
Not sure about that. Maybe rephrase as `general decoding methods, such as [Json.decodeFromString] and others`?